docs/devops-guide/quickstart.md

```diff
@@ -39,6 +44,16 @@ sudo apt-add-repository universe
 sudo apt update
 ```
 
+### OpenJDK 8
+
+* On Ubuntu:
+  Install `openjdk-8-jdk` 
+
+* On Debian either:
+   * use the .deb packages provided by: [Adoptopenjdk.org - DEB installer packages](https://adoptopenjdk.net/installation.html#linux-pkg)
+     **Note:** Install `adoptopenjdk-8-hotspot` instead of `11` of course!
+   * **or** download or build it manually (in this case use the `--no-install-recommends` flag when you install `jitsi-meet`)
```

Comment:
I don't think this is correct. Even if you set that OpenJDK would be installed because it's a depends not a suggests / recommends.

Reply:
Sry didn't see this quick review :smile: 
:thinking: Well, I don't know if thats true, because I remember using this flag for ignoring dependencies too...but I might be wrong about it.

Reply:
Ok, some internet research suggests that I remember it incorrectly.

Reply:
Still there should be a method (without needing to edit the package)...On some distros there is an option to assume that something is already installed.
Only method I see for apt is: `apt-get install bsd-mailx exim4-base-` (where exim4-base is ignored) as suggested [here](apt-get install bsd-mailx exim4-base-).

Otherwise I guess I need to remove that part from the instructions.

Reply:
I think it's best to remove it.